measures/HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -895,6 +897,18 @@ def self.get_default_hot_water_temperature(eri_version)
     return 120.0
   end
 
+  def self.get_indirect_assumed_ef()
+    return 0.95
```

Comment:
Please add a comment that this is only used for purposes of calculating tank losses. And rename it to `get_indirect_assumed_ef_for_tank_losses`.

Reply:
I also expect a `get_indirect_assumed_fuel_for_tank_losses`.